webpack2.config.js

```diff
@@ -4,7 +4,7 @@ const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const ReplaceInFileWebpackPlugin = require('replace-in-file-webpack-plugin');
 module.exports = {
   entry: {
-    lib: [
+    app: [
       path.resolve(__dirname, './src/lib/angular/angular.js'),
       path.resolve(__dirname, './src/lib/angular-animate.js'),
```

Comment:
Dado que no pudiste separarlo como querias originalmente por ahi, si lo ves util y que sirve para ordenar. podes al menos ponerle un comment (pero si ves que le agrega orden, sino como gustes):
```suggestion
      // files related to external libs (o lo que sea)
      path.resolve(__dirname, './src/lib/angular/angular.js'),
      path.resolve(__dirname, './src/lib/angular-animate.js'),
```

Reply:
Podemos explicar bien la estructura en el readme del proyecto